src/main/features/dashboard/views/index.njk

```diff
@@ -4,82 +4,67 @@
-        <table class="claim-dashboard claims">
+        <table class="claim-dashboard">
           <thead>
           <tr>
             <th scope="col">{{ t('Claim number') }}</th>
             <th scope="col">{{ t('Defendant name') }}</th>
             <th scope="col">{{ t('Claim amount') }}</th>
 
-            <th scope="col">{{ t('Status') }}</th>
+            <th scope="col" class="mobile-hide">{{ t('Status') }}</th>
           </tr>
           </thead>
           <tbody>
           {% if claimDraftSaved %}
-            <tr class="draft">
+            <tr class="draft mobile-table">
               <td><a href="{{ paths.claimTaskListPage.uri }}">{{ t('Draft claim') }}</a></td>
               <td>-</td>
 
               <td>-</td>
-              <td>{{ t('In progress') }}</td>
+              <td class="mobile-hide">
+                <a href="{{ paths.claimTaskListPage.uri }}">{{ t('Continue with claim') }}</a>
+              </td>
+            </tr>
+            <tr>
+              <td colspan="3" class="mobile-table-status">
+                {{ t('Status: ') }}<a href="{{ paths.claimTaskListPage.uri }}">{{ t('Continue with claim') }}</a>
+              </td>
             </tr>
           {% endif %}
           {% for claim in claimsAsClaimant %}
-            <tr class="claims default">
+            {% if claim.eligibleForCCJ %}
+              {% set status = t('Request a County Court Judgment.') %}
+            {% elseif claim.countyCourtJudgmentRequestedAt %}
+              {% set status =
+              t('You requested a County Court Judgment on %s. We will contact you within 5 days.',
+              { postProcess: 'sprintf', sprintf: [ claim.countyCourtJudgmentRequestedAt | date ]})
+              %}
+            {% else %}
+              {% set status = t('Your claim is being processed. We will email you to explain what to do next.') %}
+            {% endif %}
+
+
+            <tr class="claims default mobile-table">
               <td><a href="/dashboard/{{ claim.externalId }}/claimant">{{ claim.claimNumber }}</a></td>
               <td>{{ claim.claimData.defendant.name }}</td>
               <td>{{ claim.totalAmount | numeral }}</td>
-              <td>
-                {% if claim.eligibleForCCJ %}
-                  {{ t('Request a County Court Judgment.') }}
-                {% elseif claim.countyCourtJudgmentRequestedAt %}
-                  {{
-                    t('You requested a County Court Judgment on %s. We will contact you within 5 days.',
-                      { postProcess: 'sprintf', sprintf: [ claim.countyCourtJudgmentRequestedAt | date ]})
-                  }}
-                {% else %}
-                  {{ t('Your claim is being processed. We will email you to explain what to do next.') }}
-                {% endif %}
-              </td>
+              <td class="mobile-hide">{{ status }}</td>
             </tr>
+            <tr><td colspan="3" class="mobile-table-status">{{ t('Status: ') + status }}</td></tr>
```

Comment:
Shouldn't `status` be translated too?

Reply:
Oh it is translated earlier, ignore.

Reply:
Or -- what do you think about translating the status here instead of in the conditional block above?

Reply:
can't really because some stuff shouldn't get translated and each status content is quite different in regards to placeholders

Reply:
👍